README.md

```diff
@@ -70,14 +70,25 @@ class App : Application() {
    }
 }
 ```
-The Builder for CustomerIO exposes configuration options for features such `region`,`timeout`.
+The `Builder` for CustomerIO exposes configuration options for features such `region`,`timeout`.
+
+```kotlin
+        val builder = CustomerIO.Builder(
+            siteId = "YOUR-SITE-ID",
+            apiKey = "YOUR-API-KEY",
+            appContext = this
+        )
+        builder.setRegion(Region.EU)
+        builder.setTimeout(8000L)
```

Comment:
I suggest renaming the timeout function and/or making a comment about what the timeout means. Here, it's unclear what "timeout" is used for in the SDK.